.drone.yml

```diff
@@ -8,7 +8,7 @@ volumes:
 
 steps:
 - name: restore-cache
-  image: maven:3.6.3-adoptopenjdk-8
+  image: maven:3.8.4-adoptopenjdk-11
```

Comment:
is there some way to extract a variable for this image name?

Reply:
> is there some way to extract a variable for this image name?

done